packages/node_modules/@node-red/editor-client/src/js/nodes.js

```diff
@@ -2419,6 +2419,9 @@ RED.nodes = (function() {
                     if (otherNode.z === n.z) {
                         // Both ends in the same flow/subflow
                         return true
+                    } else if (n.type === "link call") {
+                        // Link call node can call out of a subflow
+                        return true
                     } else if (!!getSubflow(n.z) || !!getSubflow(otherNode.z)) {
```

Comment:
```suggestion
                    } else if (n.type === "link call" && !!getSubflow(otherNode.z)) {
                        // Link call node can call out of a subflow as long as otherNode is
                        // not in a subflow
                        return false
                    } else if (!!getSubflow(n.z) || !!getSubflow(otherNode.z)) {
```

Reply:
Ok so, a subflow can call a tab but a tab cannot call a subflow.
Looks good to me 🙂